src/ReactCompat.ts

```diff
@@ -0,0 +1,11 @@
+import { findDOMNode } from 'react-dom';
+
+export const findNodeHandle = findDOMNode;
+export const Platform = { OS: 'web' };
+export const NativeModules = {};
+export const isDEV = process.env.NODE_ENV === 'development';
+export const DeviceEventEmitter = {
+  addListener: (_a: string, _b: any) => null,
+};
+export type EmitterSubscription = any;
```

Comment:
```suggestion
export type EmitterSubscription = unknown;
```

Same

Reply:
Done in [a99ec94](https://github.com/software-mansion/react-native-gesture-handler/pull/2852/commits/a99ec94d7640da398f993bbe2e5a65a38d5844fc)